CMakeLists.txt

```diff
@@ -47,12 +51,12 @@ include_directories("$ENV{HOME}/software/spdlog/include")
 include_directories("$ENV{HOME}/software/jsoncpp/include")
 include_directories("$ENV{HOME}/software/cppkafka/include")
 
-target_link_libraries(JasmineGraph /usr/local/lib/libmetis.a)
-target_link_libraries(JasmineGraph $ENV{HOME}/software/xerces-c-3.2.2/lib/libxerces-c.so)
+#target_link_libraries(JasmineGraph /usr/local/lib/libmetis.a)
```

Comment:
We should remove these commented lines from the PR

Reply:
Commented line removed